pkg/apis/pipeline/v1alpha1/pipeline_types.go

```diff
@@ -62,6 +64,8 @@ const (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +genclient:noStatus
+// +kubebuilder:storageversion
```

Comment:
I think **this** shouldn't be here (because the storage version is `v1beta1`).

> marks this version as the "storage version" for the CRD for conversion.

Reply:
But there is no other definition of pipelines, so where would controller gen know the definition from?

Reply:
There is 2 definitons of pipeline : [`v1beta1`](https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1alpha1/pipeline_types.go) and [`v1alpha1`](https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1beta1/pipeline_types.go) definitions. Looking at the documentation of `kubebuilder:storageversion` it is to mark which version is the storage version ; in our case, the current storage version is `v1beta1`.

Reply:
Ah thank you, I didn't see the v1beta1 version, will move the storageversion comment there 👍

Reply:
I removed the overwrite, now everything is version in line with the package it's in.

I found the v1beta1 package for the pipeline type, but for none of the other CRDs, so the change impacts all CRDs generated: [`21bb334` (#3173)](https://github.com/tektoncd/pipeline/pull/3173/commits/21bb334bddfe14ab077dee7a1b3dd7c8441049de)